src/ui/store/actions/editHistory.ts

```diff
@@ -127,7 +137,7 @@ export function addEditHistoryPoint({unitId, newState}: {unitId: string; newStat
                 diff = jdp.diff(oldState, newState);
             }
 
-            dispatch(_addEditHistoryPoint({unitId, diff, state: newState}));
+            dispatch(_addEditHistoryPoint({unitId, diff, state: newState, stacked}));
```

Comment:
Can you explain why do we need stacked?

Reply:
I've tried to explain it [here](https://github.com/datalens-tech/datalens-ui/pull/1840/files#diff-c96d83998e87ddd3090121dd876da4db5e6ade7a8d782505b3cafcd3ac85d570R111-R114). Maybe this comment isn't clear